back/app/models.py

```diff
@@ -0,0 +1,61 @@
+from bson import ObjectId
+from datetime import datetime
+from app import db
+class Data:
+    def __init__(self, name, age=None, level=None, msg=None, important=False, status=None, _id=None):
+        try:
+            self.id = _id if _id else ObjectId()
+            self.name = name
+            self.age = age if age else datetime.now()
```

Comment:
datetime.now() returns more than year-month-day
please modify to match only yyyy-mm-dd format